tools/gulp/gulpfile.js

```diff
@@ -1,154 +1,206 @@
 'use strict';
 
-// http://stackoverflow.com/a/32940141
-require('es6-promise').polyfill();
-
 // Load Plugins
 // ==========================================
-var gulp = require('gulp');
-var browsersync = require('browser-sync').create();
-var minifycss = require('gulp-minify-css');
-var rename = require('gulp-rename');
-var sass = require('gulp-sass');
-var autoprefixer = require('gulp-autoprefixer');
-var concat = require('gulp-concat');
-var uglify = require('gulp-uglify');
-var imagemin = require('gulp-imagemin');
-var cache = require('gulp-cache');
-var debug = require('gulp-debug');
-var validate = require('gulp-jsvalidate');
+const gulp = require('gulp');
+const bytediff = require('gulp-bytediff');
```

Comment:
It's not a change, rather browsersync is being removed because the dev environment doesn't support it anyway, and bytediff was added to do something different (print minification statistics).